docs/articles/baseplatform/creating-a-new-layer.rst

```diff
@@ -182,3 +182,18 @@ won't need to modify all the recipes.
 
 .. [#bbmanual] http://www.yoctoproject.org/docs/latest/bitbake-user-manual/bitbake-user-manual.html
 .. [#templateconf] http://www.yoctoproject.org/docs/latest/dev-manual/dev-manual.html#creating-a-custom-template-configuration-directory
```

Comment:
I think all the footnotes should go in the bottom of the page.

Reply:
fixed